### DISCUSSION TITLE
Trane T400 stage two heat and cool

### DISCUSSION THREAD

**Discussion by dinki** [zwave-js/zwave-js#6574] [General]
Hey guys. I've been using a Trane T400 thermostat for many many years and it is great. This summer my a/c unit died and I replaced it with a new unit that has dual stage cool and heat. These modes are displayed on the thermostat display but I don't get those modes reported by zwave-js. I'm not sure if this is because it is not supported or just not been added. 

Is there a way for me to see all zwave messages being sent from e thermostat to determine if it exists?  I'm more than willing to try to add this myself but don't know how to snoop the messages. Thanks

**Comment by AlCalzone** [comment]
Please fill out [this issue template](https://github.com/zwave-js/node-zwave-js/discussions/new?category=request-support-investigate-issue) and provide the requested information, especially the log of a re-interview. That should tell us if and which action is necessary.
